lib/spack/spack/schema/cdash.py

```diff
@@ -14,14 +14,28 @@
     "cdash": {
```

Comment:
@kwryankrattiger Where else is this used? I saw you updating this file in the change history.  Are you willing to review? We'd like to be able to set our cdash information in the environment manifest rather than having to pipe it to the CLI.

Reply:
We use this in the `lib/spack/spack/ci.py` module for the `CDashHandler`.